debug.log

```diff
@@ -0,0 +1,3 @@
+[1113/095750.014:ERROR:process_reader_win.cc(151)] SuspendThread: Acesso negado. (0x5)
```

Comment:
Remove file and add it to ;gitignore. Which tool are generating this debug fiile?

Reply:
I don't know.